app/services/payment_provider_customers/stripe_service.rb

```diff
@@ -86,6 +104,15 @@ def api_key
       organization.stripe_payment_provider.secret_key
     end
 
+    def checkout_link_params
+      {
+        success_url: 'https://www.getlago.com',
```

Comment:
It's the URL customer will be redirected to after checkout success.

Indeed that's odd and for now, we agreed with product to do so until complaints. And for now, we do not have an existing constant for such or other occurrences in the app.
Maybe it should be defined on top of the file with an explicit constant correct?

Reply:
Are you ok with letting it like this today? I assume later we would like to add a success page on either the app, the website or s3 (static file) to improve experience

Reply:
If it's intended then fine with me :)

Well ideally I think it could be a setting somewhere but meanwhile maybe turning it into a constant is a good first step.

Reply:
I've set it in the class on top of the file. Tested and it works